supervisor/internal/examples/config_sample.yaml

```diff
@@ -0,0 +1,78 @@
+# Connection details for the OpAMP server.
+server:
+    endpoint: wss://localhost:1729
+    #    tls:
+    # Client-side TLS configuration to connect to the OpAMP server.
+
+# Watchdog allows the supervisor to monitor the health of the agent and ensure
+# it is always running (restarting if necessary). In this sense, it acts as
+# an alternative to agent installations that are not managed by systemd.
+#
+# A watchdog policy, like `systemd', albeit not as exhaustive, allows the
+# watchdog to honor certain timing restrictions during the start, stop and
+# restart of the agents.
+watchdog:
+    # Command describes how the agent must be exec'd.
+    command:
+        # Full or relative path to the agent executable.
+        name: /opt/otcol/bin/otel
+        # Set of command-line arguments required to start the `executable`
+        # specified above.
+        args:
+            - --config
+            - /etc/otcol/config.yaml
```

Comment:
It is a pity that we need to specify the config file location twice. It is probably inevitable for a generic supervisor. For Otel Collector specific Supervisor we should avoid doing this.

Reply:
How can you avoid it? An assumed default path?

Reply:
I mean avoid specifying it twice. The path to the config file is also specified on line 68.

If the Supervisor knows that it deals with Otel Collector then we don't need to tell the Supervisor how to pass the path to the config file on the command line using `args` section, the Supervisor can copy the value from line 68 into the command line argument `--config` automatically.

Because this is a generic Supervisor it does not have this knowledge about Otel Collector and we have to have an `args` section as well.